recipes/sisl/run_test.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+# Test command:
+pytest --pyargs sisl
```

Comment:
If you'd rather not have just one line in a separate file, you could just put this in the yaml file like
```
test:
  commands:
    - pytest --pyargs sisl
```
No need to if you'd prefer it separate though.

Reply:
I probably prefer the other thing (then I can expand in the script later).